### PR TITLE
Fix light emitters being deleted in pod bay when deleting everything

### LINKED ISSUE #83219: Clearing a pod bay with the supply pod config deletes the lighting
## Issue Summary

Using the "wipe bay" button clears out the lights in the selected pod bay.

image

## Round ID:

[228811](https://scrubby.melonmesa.com/round/228811)

## Testmerges:

[#82298: 'Wawastation' by mc-oofert at commit 02a7de55ef](https://www.github.com/tgstation/tgstation/pull/82298)
[#83163: '[Icebox] Retvrn: Brings Bar Back to Base (level)' by MrMelbert at commit 638ec5057b](https://www.github.com/tgstation/tgstation/pull/83163)
[#83175: 'Adds Minecarts' by MrMelbert at commit 9aa850575e](https://www.github.com/tgstation/tgstation/pull/83175)

## Reproduction:

Use Config/Launch Supplypod verb.

Press the trash button.

image

The bay will no longer be lit.

### PR DESCRIPTION
**Pull request by Jewelry-x**
## About The Pull Request

When you delete everything with the "delete that shit" button in the pod bay, it also deleted the light emitters, which makes it difficult to see what is happening in the bay. This PR makes it so the light emitters are not deleted. Also removes some one letter variables in that part of the code.

* Closes https://github.com/tgstation/tgstation/issues/83219
## Why It's Good For The Game

Helps admins do their work more easily.
## Changelog
:cl:
code: removed two one letter variables
admin: clearing the pod bay will not remove the light emitters.
/:cl:

### CODE REVIEW

File: code/modules/cargo/centcom_podlauncher.dm [hunk 1]
```diff
@@ -788,12 +788,16 @@ ADMIN_VERB(centcom_podlauncher, R_ADMIN, "Config/Launch Supplypod", "Configure a
 		selector.moveToNullspace() //Otherwise, we move the selector to nullspace until it is needed again
 
 /datum/centcom_podlauncher/proc/clearBay() //Clear all objs and mobs from the selected bay
-	for (var/obj/O in bay.get_all_contents())
-		qdel(O)
-	for (var/mob/M in bay.get_all_contents())
-		qdel(M)
+	for (var/obj/object in bay.get_all_contents())
+		if (object.type == /obj/effect/light_emitter/podbay)
```
Comment (SmArtKar) [inline code/modules/cargo/centcom_podlauncher.dm:792]:
```suggestion
		if (istype(object.type, /obj/effect/light_emitter/podbay))
```
don't compare types directly

File: code/modules/cargo/centcom_podlauncher.dm [hunk 2]
```diff
@@ -788,12 +788,16 @@ ADMIN_VERB(centcom_podlauncher, R_ADMIN, "Config/Launch Supplypod", "Configure a
...
+			continue
+		qdel(object)
+	for (var/mob/mob in bay.get_all_contents())
+		qdel(mob)
 	for (var/bayturf in bay)
 		var/turf/turf_to_clear = bayturf
+		if (turf_to_clear.type == /obj/effect/light_emitter/podbay)
```
Comment (SmArtKar) [inline code/modules/cargo/centcom_podlauncher.dm:799]:
```suggestion
		if (istype(turf_to_clear, /obj/effect/light_emitter/podbay))
```

**Review by SmArtKar — CHANGES_REQUESTED** [review]

File: code/modules/cargo/centcom_podlauncher.dm (hunk 2 above)
Comment (Jewelry-x) [reply code/modules/cargo/centcom_podlauncher.dm]: my bad, fixed both

**Comment by mc-oofert** [comment]
maybe this shouldn't delete any effects instead of that particular object

**Comment by Jewelry-x** [comment]
> maybe this shouldn't delete any effects instead of that particular object

Isn't there a case where an admin would like to delete unwanted effects?

**Pull request merged**